src/Form.vue

```diff
@@ -88,10 +88,18 @@
         &__checkboxes,
         &__checkbox,
         &__radios,
+        &__switch,
         &__date {
             display: block;
             margin: 2em 0;
 
+        &__checkboxes,
+        &__switch,
+        &__radios {
+            label > span {
+                display: inline;
+            }
+
```

Comment:
I think this code is coming from a merge conflict. Only the change on line 91 is needed.